chrome-ext/script.js

```diff
@@ -0,0 +1,26 @@
+
+
+const main = document.querySelector("main")
+
+main.style.backgroundImage = "url(https://source.unsplash.com/random/1920x1080)"
```

Comment:
The background image size should be responsive. Ideally, the background image dimensions should be the closest to the device it is being used in. For e.g. A MacBook Air has a 1440 x 900 pixel resolution screen.

Reply:
yeah... my pc is 1920x1080... but the i realize i should have added width 100% or, position center so i don't loose the picture when shrinking the screen